programs-remote-listings/views/admin-settings.php

```diff
@@ -9,16 +9,28 @@
     <form action="options.php" method="post"><?php
         settings_fields('rs_remote_settings');
         do_settings_sections(__FILE__);
-        $options = get_option('rs_remote_settings'); ?>
+        $options = get_option('rs_remote_settings');
+        $rs_domain = (! empty($options['rs_domain']) && $options['rs_domain'] != '') ? $options['rs_domain'] : '';
+        $base_domain = '.secure.retreat.guru';
+        $http = 'https://';
+
+        if (isset($_SERVER['SERVER_NAME']) && ('rgconnect.test' == $_SERVER['SERVER_NAME'] || 'qa-testing.booking-demo.com' == $_SERVER['SERVER_NAME'])) {
```

Comment:
This exposes internal development and QA URLs and locations to the public, which is generally ill-advised.  It's low-risk, but given our lower level of security and password entropy on our QA environments (and propensity to import live customer data), perhaps there is a better way around this problem (maybe looking for a defined constant which allows for a custom url).  Happy to jam further with you on this

Reply:
I like that idea. I was doing some stuff with constants to get around testing issues.
But that is probably a good way of doing it.

Reply:
see env comment above